src/App.js

```diff
@@ -23,6 +25,9 @@ function App() {
         <Route path = "/signup" element = {<Signup />} />
         <Route path = "/watch-later" element = {<WatchLaterPage />} />
         <Route path = "/liked-videos" element = {<LikedVideos />} />
+        <Route path = "/playlists" element ={<PlaylistListing />} />
+        <Route path = "/playlists/:videoId" element ={<PlaylistListing />} />
+        <Route path = "/playlist/:playlistId" element = {<PlaylistPage />} />

```

Comment:
you can nest playlists/:videoId route inside playlists route.
You can read more about it here  [https://reactrouter.com/docs/en/v6/getting-started/overview#nested-routes](url)